index.js

```diff
@@ -104,20 +94,16 @@ module.exports = opts => {
       return cb(new PluginError('gulp-html', 'Streaming not supported'));
     }
 
-    vnuArgs = [...vnuArgs, file.history];
+    vnuArgs.push(file.history);
```

Comment:
I wonder if this should be `vnuArgs.push(...file.history)`? Now it's added as an array in array:

```
[
  '-Xss1024k',
  '-jar',
  '"C:\\Users\\xmr\\Desktop\\gulp-html\\node_modules\\vnu-jar\\build\\dist\\vnu.jar"',
  [ 'C:\\Users\\xmr\\Desktop\\gulp-html\\test\\ng.html' ]
]
```